docs/xxusegcstartuphints.md

```diff
@@ -50,7 +50,7 @@ The final value stored to the shared cache is not recorded in the verbose GC out
 
 :fontawesome-solid-pencil:{: .note aria-hidden="true"} **Notes:**
 
-- When enabled, this option overrides any initial heap size that is specified on the command line, for example by using the [`-Xms`](xms.md) option.
+- When enabled, this option overrides any initial heap size that is specified on the command line while considering the limits set by that command-line option. For example, if the initial heap size is set by using the [`-Xms`](xms.md) option, then the new initial heap size will be greater than the heap size set by the `-Xms` option.
```

Comment:
(-Xms, Xmx) limits do not have to be explicitly set by that command-line option. They could be implicitly inferred. Regardless if they are implicit or explicit, the new initial value is within those limits.

Reply:
also not sure about 'the limits set by that command-line option' part, again since limits do not have to be set explicitly.

Reply:
This was the existing content, so I didn't change it. I will see what I can do. Thanks!